src/components/button/_button.scss

```diff
@@ -201,6 +201,10 @@
     }
   }
 
+  .#{$prefix}--btn:not(.#{$prefix}--btn--ghost) {
+    padding-right: rem(64px);
```

Comment:
instead of adding this class which will override the existing styles we can just modify the `$btn-padding` and `$btn-padding-sm` variables. other than that, looks good to me!